DataFormats/Detectors/TPC/include/DataFormatsTPC/TrackTPC.h

```diff
@@ -110,6 +111,9 @@ class TrackTPC : public o2::track::TrackParCov
     uint8_t sectorIndex, rowIndex;
     return (getCluster(nCluster, clusters, sectorIndex, rowIndex));
   }
+  const dEdxInfo& getdEdx() const { return mdEdx; }
+  dEdxInfo getdEdx() { return mdEdx; }
```

Comment:
why do you need both versions? Shouldn't the second also be const since effectively a copy will be returned?

Reply:
Yes, we need both, but there is a typo in the second. The two should return const ref / ref. Will fix that.

Reply:
changed my opinion, the reconstruction will create the dEdx and then it should be used but not changed, so I removed the non-const getter. The user can create a copy if needed.